### PR TITLE
fixed opticon button styles/spacing/footer

### PR DESCRIPTION
**Pull request by julesforrest**
@stewsmith Navbar button smaller, padding on blue background headlines more even, spacing between sections fixed, distance from track icons fixed, text tightened up

### CODE REVIEW

**Comment by dtothefp** [comment]
@julesforrest is this ready for review? I'm deploy captain this week so let me know

**Comment by stewsmith** [comment]
@dtothefp @julesforrest I worked with Jules on this PR so I'll review.

File: website-guts/assets/css/modules/opticon-header.scss [hunk 1]
```diff
@@ -17,9 +17,13 @@
       text-decoration: none;
     }
   }
-  .body-button {
+  .body-button--nav {
     margin: 0;
     color: $opticon-brand-blue;
+    font-size: 0.87em;
+    padding: 5px 17px;
+    border: 1px solid $brand-blue;
```
Comment (stewsmith) [inline website-guts/assets/css/modules/opticon-header.scss]: please use `$opticon-brand-blue`. They're actually the same, but we'll have more flexibility if we want to change the opticon colors without changing the Optimizely brand colors.

File: website-guts/assets/css/pages/opticon-homepage.scss [hunk 2]
```diff
@@ -89,14 +88,21 @@ body.opticon-homepage {
...
 
+
 @media only screen and (min-width: $desktop-min-width) {
   body.opticon-homepage {
     .description {
       padding: 5% 0;
       width: 600px;
     }
+    .track-container {
+      padding: 115px 10px 30px 10px;
```
Comment (stewsmith) [inline website-guts/assets/css/pages/opticon-homepage.scss]: This looks the same as the old page. Are you sure this rule is being applied?

**Comment by stewsmith** [comment]
@julesforrest I cleaned up some CSS in this branch and made the button styles a little more modular.

File: website-guts/assets/css/pages/opticon-homepage.scss (hunk 2 above)
Comment (julesforrest) [inline website-guts/assets/css/pages/opticon-homepage.scss]: It looks like it's being applied on my end

**Comment by julesforrest** [comment]
@stewsmith Cool sounds good

**Comment by stewsmith** [comment]
retest this please

File: website-guts/assets/css/pages/opticon-homepage.scss [hunk 3]
```diff
@@ -89,14 +88,21 @@ body.opticon-homepage {
     .page-title {
       font-size: 4em;
     }
+    .register-title {
+      width: 50%;
+    }
```
Comment (stewsmith) [inline website-guts/assets/css/pages/opticon-homepage.scss]: Can you put this in `website-guts/assets/css/layouts/opticon.scss` so it applies to every register banner?

Comment (julesforrest) [inline website-guts/assets/css/pages/opticon-homepage.scss]: Yup added that @stewsmith 

(@jeffz-optimizely)

**Comment by stewsmith** [comment]
LGTM

**Pull request merged**